README.rst

```diff
@@ -61,11 +61,13 @@ extraction code from a spider:
 
 
     class BooksSpider(scrapy.Spider):
+
         name = 'books'
         start_urls = ['http://books.toscrape.com/']
 
         def parse(self, response):
-            for url in response.css('.image_container a::attr(href)').getall():
+            query = '.image_container a::attr(href)'
+            for url in response.css(query).getall():
                 yield response.follow(url, self.parse_book)
```

Comment:
It can be shortened further to

`yield from response.follow_all(css='.image_container a', callback=self.parse_book)`

or 

```py
book_links = response.css(".image_container a")
yield from response.follow_all(book_links, self.parse_book)
```

Reply:
Done